src/discord-cluster-manager/leaderboard_db.py

```diff
@@ -743,6 +743,26 @@ def get_leaderboard_submission_count(
         self.cursor.execute(query, args)
         return self.cursor.fetchone()[0]
 
+    def create_user_from_cli(self, user_id: str, user_name: str, cli_id: str):
+        """
+        Method to create a user from the CLI. Shouldn't be used for Discord.
+        """
+        try:
+            self.cursor.execute(
+                """
+                INSERT INTO leaderboard.user_info (id, user_name, cli_id)
+                VALUES (%s, %s, %s)
+                ON CONFLICT (id) DO UPDATE
+                SET user_name = %s, cli_id = %s
+                """,
+                (user_id, user_name, cli_id, user_name, cli_id),
+            )
+            self.connection.commit()
+        except psycopg2.Error as e:
+            self.connection.rollback()
+            logger.exception("Could not create/update user %s from CLI.", user_id, exc_info=e)
+            raise KernelBotError(f"Could not create/update user {user_id} from CLI.") from e
```

Comment:
KernelBotError is intended to convey error messages to the end user, not exposing any internal data (that we'd log with logger). 
AFAICS, this just triggers `raise HTTPException(status_code=400, detail="Failed to create user")`, so just re-raising the psycopg2 error would be fine here?